insolar/assured-ledger
# PLAT-387: Add VFindCallRequest/VFindCallResponse messages

**- What I did**
Add a new messages in insolar/payload/proto_vnvn.r0.proto
VFindCallRequest/VFindCallResponse 

**- How I did it**

**- How to verify it**
run:
`make generate-protobuf`
And check that you do not have any changes in git stage

**- Description for the changelog**

https://insolar.atlassian.net/browse/PLAT-387
